test/README.md

```diff
@@ -10,10 +10,10 @@ But don't give up hope!!! Although our tests may appear complex and overwhelming
 
 * To run an individual suite: (recommended during development for easier isolated diffs)
 
-Example: `$(npm bin)/mocha --grep ConfigTestCases`
+Example: `$(npm bin)/jest --watch ConfigTestCases`
```

Comment:
Could you also replace `$(npm bin)/` by `yarn`?

```
yarn jest --watch ConfigTestCases
```

Reply:
```
yarn jest --testPathPattern ConfigTestCases
```

Reply:
`--testPathPattern` is not required.
`yarn jest ConfigTestCases` only runs `ConfigTestCases` as expected.